tarofreighter/chain_porter.go

```diff
@@ -265,20 +265,10 @@ func (p *ChainPorter) waitForPkgConfirmation(pkg *OutboundParcelDelta) {
 
 	log.Infof("Waiting for confirmation of transfer_txid=%v", txHash)
 
-	// Before we can broadcast, we want to find out the current height to
-	// pass as a height hint.
-	ctx, cancel := p.WithCtxQuit()
-	defer cancel()
-	currentHeight, err := p.cfg.ChainBridge.CurrentHeight(ctx)
-	if err != nil {
-		p.cfg.ErrChan <- mkErr("unable to get current height: %v", err)
-		return
-	}
-
 	confCtx, confCancel := p.WithCtxQuitNoTimeout()
 	confNtfn, errChan, err := p.cfg.ChainBridge.RegisterConfirmationsNtfn(
 		confCtx, &txHash, pkg.AnchorTx.TxOut[0].PkScript, 1,
-		currentHeight, true,
```

Comment:
Ah ok, yeh this itself is the root cause. We should also check the asset minting and other relevant locations to make sure we're not running into the same issue.

Reply:
Yeah, that is true. I see a few other confirmation registrations that could have the same issue.

In that case would it perhaps make sense to keep the `height_hint` in the `chain_txns` table, as it would tie each on-chain tx to a "first possible block"?

Reply:
> In that case would it perhaps make sense to keep the height_hint in the chain_txns table, as it would tie each on-chain tx to a "first possible block"?

Yeah good point. I think rn, in most cases we're not really looking for a spend for an outpoint where we might not actually have the transaction itself.

Reply:
Also added for the other confirmation registrations.